tests/test_mesh.py

```diff
@@ -137,11 +234,72 @@ def check_texture(tm_color, tm_texture, gs_texture, dim, material_name, texture_
         if tm_material.emissiveFactor is None and tm_material.emissiveFactor is None:
             assert gs_material.emissive_texture is None
         else:
-            check_texture(
+            check_gs_tm_textures(
+                gs_material.emissive_texture,
                 tm_material.emissiveFactor,
                 np.array(tm_material.emissiveTexture),
-                gs_material.emissive_texture,
+                0.0,
                 3,
                 material_name,
                 "emissive",
             )
+
+
+@pytest.mark.required
+@pytest.mark.xdist_group(name="huggingface_hub")
+@pytest.mark.parametrize("usd_filename", ["usd/sneaker_airforce", "usd/RoughnessTest"])
+def test_usd_parse(usd_filename):
+    glb_folder = snapshot_download(
```

Comment:
We do not use `snapshot_download` and `@pytest.mark.xdist_group(name="huggingface_hub")` anymore